lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3415,7 +3415,7 @@ def setUpClass(cls):
         test_subject = u'\u1234 test subject'
         test_message = u'\u6824 test message'
         cls.full_test_message = {
-            'send_to': 'staff',
+            'send_to': '["myself", "staff"]',
```

Comment:
This `full_test_message` variable is used in several tests, and had to be converted to a single-element array anyways, so I elected to add another value to more adequately test "the API can handle multiple `send_to` values" during those tests.